CHANGELOG.md

```diff
@@ -3,6 +3,76 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.4.0
+
+_Jan 28, 2022_
+
+A big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚣 Introduce [variable row height](https://mui.com/components/data-grid/rows/#variable-row-height) (#438) @DanailH
+
+Allows for setting a row-specific height. Before all rows had the same height, now you can set the height on a per-row basis.
+
+```tsx
+<DataGrid 
+  getRowHeight={
+    ({ id }: GridRowHeightParams) => (id % 2 === 0 ? 100 : null)
+  }
+/>
+```
```

Comment:
```suggestion
  Allows for setting a row-specific height. Before all rows had the same height, now you can set the height on a per-row basis.
  
  ```tsx
  <DataGrid 
    getRowHeight={
      ({ id }: GridRowHeightParams) => (id % 2 === 0 ? 100 : null)
    }
  />
```
```

Move it under bullet point for better alignment?